_posts/2018/2018-09-02-careful-with-present-when-using-activerecord-scopes.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Careful with present? when using ActiveRecord scopes
+description: 'Mostly it’s fine, until your app goes boom.'
+layout: article
+category: ruby
+image:
+  base: '2018/careful-with-present-when-using-activerecord-scopes'
+  alt: 'Stadtbibliothek, Stuttgart, Deutschland'
+  source: 'https://unsplash.com/photos/PkbZahEG2Ng'
+  credit: 'Tobias Fischer'
+---
+
+When using Rails, we tend to use `#present?` in many places as it is added to _all_ Ruby objects by Rails. For example, you might use it in a view to check if an array has some elements before trying to loop through it and display some data.
```

Comment:
I would say: as it is available on all Ruby objects when you're using Rails (or something like that)